src/Bijectors.jl

```diff
@@ -243,8 +243,6 @@ end
 
 include("interface.jl")
 
-# Broadcasting here breaks Tracker for some reason
-maporbroadcast(f, x::AbstractArray{<:Any, N}...) where {N} = map(f, x...)
```

Comment:
I still think it would make sense to keep this definition (and only change it for ReverseDiff). If the arrays are of the same dimension, it should be simpler to not invoke the broadcast machinery (for which adjoints require quite complex implementations e.g. in Tracker and Zygote that even make use of ForwardDiff).

Reply:
Tracker's map also reduces to an array of TrackedReals. So broadcasting by default is needed for 2 out of 3 AD backends. So it makes sense as the default. Only when we have mixed TrackedArrays and arrays of TrackedReals that Tracker fails hence the need for the map workaround. ReverseDiff broadcasting doesn't fail in this case.